tools/config-from-doc.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
I see that behavior implemented here and there and I'm not sure to like it. What if I want to grep some variables to make my own preset? Or sort the lines, automate the enabling of all features, ...?

Wouldn't it be a better compromise to print a warning message on stderr and let the script continue, just in case?

Reply:
This test only checks if `/dev/stdout` is a terminal. If you pipe the output to another command, it will also fail:
```bash
$ ./tools/external-tool-tester.sh | less
---Uname:
...
```
So I disagree. You could add some sort of argument that could be passed if someone really wanted to print to the terminal, but it would be just as easy to pipe to `less`

Reply:
I see. This is a good-enough compromise, then.